src/backend/marsha/core/views.py

```diff
@@ -254,6 +255,7 @@ def _get_app_data(self, cache_key, lti=None, resource_id=None):
                             "can_return_live_info": lti.is_admin or lti.is_instructor
                             if lti
                             else False,
+                            "request": self.request,
```

Comment:
Why do you need this ? I see no usage.

Reply:
Good catch!
It may come from an experiment, it is useless now.

Reply:
After getting back to BBB integration, I found that it is needed to build an url from a serializer.
See get_lti_url in src/backend/marsha/bbb/serializers.py https://github.com/openfun/marsha/pull/1107/files#diff-40b20cc0ee1773c08a7fbfc9aa3bf7e3ec0c2f91a80770fa3eafa32d38115387R50

Reply:
Please ignore my last comment, as I managed to get rid of this.